diesel/src/sql_types/ord.rs

```diff
@@ -0,0 +1,19 @@
+use sql_types::{self, NotNull};
+
+/// Marker trait for types which can be used with `MAX` and `MIN`
+pub trait SqlOrd {}
+
+impl SqlOrd for sql_types::SmallInt {}
+impl SqlOrd for sql_types::Integer {}
+impl SqlOrd for sql_types::BigInt {}
+impl SqlOrd for sql_types::Float {}
+impl SqlOrd for sql_types::Double {}
+impl SqlOrd for sql_types::Text {}
+impl SqlOrd for sql_types::Date {}
+impl SqlOrd for sql_types::Interval {}
+impl SqlOrd for sql_types::Time {}
+impl SqlOrd for sql_types::Timestamp {}
+impl<T: SqlOrd + NotNull> SqlOrd for sql_types::Nullable<T> {}
+
+#[cfg(feature = "postgres")]
```

Comment:
All other impl in `sql_types` are unconditional to the enabled backends, so I think this should also be implemented in every case.

Reply:
The `Array` type doesn't exist unless this feature is enabled.

Reply:
Then maybe enable it also unconditionally?
(It seems a bit strange that this type is only there if "postgres" is one of the enabled backend)